components/landing/ContentBlock/LeftContentBlock/index.js

```diff
@@ -1,17 +1,23 @@
-import React, { useState, useCallback } from 'react';
+import React, { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import Router from 'next/router';
 import PropTypes from 'prop-types';
 import { Row, Col } from 'antd';
 import Fade from 'react-reveal/Fade';
 
+import { meSelector } from '~/redux/selectors/common/user';
 import Image from '~/components/common/Image';
 
 import * as S from './style';
 
 const LeftContentBlock = ({ title, content, imgName, firstBlock }) => {
   // 임시로 작성한 state와 function
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
-  const onSubmit = useCallback(() => {
-    setIsLoggedIn(true);
+  const me = useSelector(meSelector);
+
+  const onSubmit = useCallback(e => {
```

Comment:
```suggestion
  const onSubmit = useCallback((e) => {
```
단일 인자도 () 로 감싸지 않기로 했었나요?
eslint off 설정인가요?

Reply:
eslint에서 off되어 있습니다.

Reply:
감싸는게 좋다고 생각되는데 그 날 회의때 지용님도 감싸는걸로 결정하셨었나요?

Reply:
저는 감싸는게 더 좋아보여요.